v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/dlq/BigQueryDeadLetterQueueOptions.java

```diff
@@ -22,10 +22,12 @@
 public interface BigQueryDeadLetterQueueOptions extends PipelineOptions {
   String GROUP_NAME = "Dead Letter Queue";
 
+  // Make this non-optional. Otherwise, the UI will show this as Optional Dead Letter Queue
+  // Parameters
```

Comment:
nit: Unnecessary line break?

Reply:
I will fix this in the PR - https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/1701